### ISSUE TITLE
Content of the page get truncated after applying text spacing on the page.

### ISSUE THREAD

**Issue by tyveille** [microsoft/fluentui#18103]
### Environment Information

OS Build: Version 20H2(19042.867)
Browser: Version 89.0.4389.90 (Official Build) (64-bit)
Tool: Stylus text spacing tool.

### Describe the issue:

User Experience
If content get truncated after applying text spacing on the page than low vision user not able to see all the content present on the page.
Note
User credentials should NOT be included in the bug.

MAS Reference
MAS 1.4.12 - Text Spacing ***

### Please provide a reproduction of the issue in a codepen:
 
1. Open URL: use the website
2.  Navigate to left pane and select the pane, Navigation pane will get expanded, navigate to all options   present under left Nav pane.
3.  Observe and verify the issue.

#### Actual behavior:

Content of the page get truncated after applying text spacing on the page.

#### Expected behavior:

All content should be present and accessible after applying text spacing on the page.

**Comment by ecraig12345** [comment]
Thanks for filing an issue, but I'm not sure what you mean by "applying text spacing" or which website page you're starting from. Can you please provide more details about that?

**Comment by ecraig12345** [comment]
Oh never mind, I figured it out -- https://add0n.com/stylus.html -- but in general it's helpful if you can include links to the tools or pages you're referencing.

**Comment by tyveille** [comment]
Sorry about that as I was nervous to violate the "Do not link to, screenshot or reference a Microsoft product in this description." that is in the template. I figured I would under share and then provide additional details if needed.

**Comment by ecraig12345** [comment]
@tyveille Definitely appreciate that you paid attention to that (not everybody does)! Using the website as a repro is a good way to get around that issue. But it's helpful if you can link to the specific page you were using, and maybe include a screenshot.

Just to be clear, is this issue specific to the website, or do you see it while using a particular Fluent control in some other context too?

(For future reference: assuming you're a Microsoft employee, if it's something where you really have to share internal info, you can [link your github account](https://repos.opensource.microsoft.com/link) and then use [this page](https://repos.opensource.microsoft.com/people) or [this extension](https://docs.opensource.microsoft.com/tools/browser.html) to find the real name of whoever the issue is assigned to. But an isolated repro is still preferable if at all possible.)

**Comment by tyveille** [comment]
Before Spacing:
![image](https://user-images.githubusercontent.com/22622621/119526604-624b8e80-bd34-11eb-92f6-27f9b8ccf868.png)

After Spacing is applied:
![image](https://user-images.githubusercontent.com/22622621/119526670-71cad780-bd34-11eb-83a0-3a88eced5415.png)

Sometimes the chevron is completely hidden when expanded.

**Comment by gouttierre** [comment]
@tyveille - thanks for providing the extra information. May i ask you to confirm the Fluent version you are viewing in the site? There are various versions. but you can confirm as shown below.
![image](https://user-images.githubusercontent.com/78806055/121559800-378d5580-ca17-11eb-866c-8a77839f1d32.png)

@miroslavstastny - proving the version is confirmed would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley

**Comment by tyveille** [comment]
I was able to reproduce this in v8.17.4

![image](https://user-images.githubusercontent.com/22622621/121567783-d2644080-c9d3-11eb-9fcc-38eeb3c6956e.png)

**Comment by ecraig12345** [comment]
Looks like this is in the legacy demo app (`@fluentui/react-docsite-components`, formerly `@uifabric/example-app-base`) which you can see by picking a version from https://aka.ms/fluentdemo. PR #18529 should fix the issue (no further investigation needed).
